tars/adapter.go

```diff
@@ -72,7 +65,11 @@ func (c *AdapterProxy) Recv(pkg []byte) {
 	if ok {
 		ch := chIF.(chan *requestf.ResponsePacket)
 		TLOG.Debug("IN:", packet)
-		ch <- &packet
+		select {
+		case ch <- &packet:
```

Comment:
when the packet return before the msg subscibe，
the channel will be closed and readCh will return nil to msg。

Reply:
one more pr https://github.com/TarsCloud/TarsGo/pull/73